buildpipeline/pipeline.json

```diff
@@ -79,7 +79,8 @@
             "PB_DockerTag": "ubuntu-16.04-cross-arm64-a3ae44b-20180315221921",
             "PB_Architecture": "arm64",
             "PB_BuildArguments": "-buildArch=arm64 -$(PB_ConfigurationGroup) -stripSymbols -- /p:StabilizePackageVersion=$(PB_IsStable) /p:PackageVersionStamp=$(PB_VersionStamp)",
-            "PB_SyncArguments": "-p -- /p:ArchGroup=arm64 /p:DotNetRestoreSources=$(PB_RestoreSource) /p:DotNetAssetRootUrl=$(PB_AssetRootUrl)"
+            "PB_BuildManagedArguments": "-BuildPackages=false -buildArch=x64 -$(PB_ConfigurationGroup) -stripSymbols -- /p:StabilizePackageVersion=$(PB_IsStable) /p:PackageVersionStamp=$(PB_VersionStamp)",
```

Comment:
@sdmaclea is there a reason you put arch as x64 for buildmanaged? instead of arm64?

Reply:
arm64 sdk is not available, so it cannot build if the managed arch is arm64

Reply:
Also the corex managed pieces are all published to AnyCpu or something like that.  So there should be no CPU specific code.

Reply:
Unfortunately we do compile against System.Private.CoreLib which is arch specific and so we are linking against x64 System.Private.CoreLIb. While that might not matter in the end it does show up in some of our offline builds where we weren't expecting a restore to happen in this step but instead in our sync step.

Reply:
I just changed this to arm64, https://github.com/dotnet/corefx/commit/4036846ceb1e124158e68f0cde26aa3fab524b71 before I saw your message, but if what you say is true we might fail and need to fix in a different way.

Reply:
However the arch for the sdk we use is selected in init-tools.sh and I don't believe it is influenced by the buildarch, so perhaps the right thing will still happen.

Reply:
It may start working now that there are published myget packages....